schemas/2019-11-01/Microsoft.Network.json

```diff
@@ -4,6 +4,152 @@
   "title": "Microsoft.Network",
   "description": "Microsoft Network Resource Types",
   "resourceDefinitions": {
+    "NetworkExperimentProfiles": {
```

Comment:
Since the generation of this schema file is managed by two independent teams, I'm wondering if it's worth setting a precedence by splitting into two files to match the team structure. We've done this for other RPs that are managed similarly - see the following for example:
https://github.com/Azure/azure-resource-manager-schemas/blob/da686787b8afb7b8d9b65871fd1100fd07adc8eb/generator/whitelist.ts#L177-L181
which is used to generate:
https://github.com/Azure/azure-resource-manager-schemas/blob/da686787b8afb7b8d9b65871fd1100fd07adc8eb/schemas/2020-01-01/Microsoft.Devices.Provisioning.json

If for example we call this `Microsoft.Network.Frontdoor`, then we don't need to worry about either teams overwriting each others' changes. Thoughts?

Reply:
That sounds great. I did not know this feature existed. I assume this will unblock onboarding to auto-generation for Frontdoor resources.

Reply:
Yup, feel free to onboard as part of this PR if that's helpful for your team - instructions are here:
https://github.com/Azure/azure-resource-manager-schemas/tree/master/generator#onboarding-teams-to-autogeneration-pipeline

Reply:
Thanks. I have included the changes to onboard to auto-generation.